apps/crn-frontend/src/tags/__tests__/TagsList.test.tsx

```diff
@@ -15,7 +15,7 @@ const renderTags = async (): Promise<RenderResult> =>
           <Suspense fallback="Loading...">
             <MemoryRouter initialEntries={['/']}>
               <Route path="/">
-                <TagList />
+                <TagList entities={['research-output', 'user']} />
```

Comment:
Could we export this from the file so the test cases always match the use

Reply:
committed here [review: use same entities in the test as the ones used in the non-test files](https://github.com/yldio/asap-hub/pull/3856/commits/5dcaaeaffc9e9f8746237ea4755b2c853549ec6d)